E/Eigen/build_tarballs.jl

```diff
@@ -27,7 +27,7 @@ install_license COPYING.*
 
 # These are the platforms we will build for by default, unless further
 # platforms are passed in on the command line
-platforms = supported_platforms()
+platforms = supported_platforms(; experimental=true)
```

Comment:
No, there is no need to do this
```suggestion
platforms = [AnyPlatform()]
```